index.html

```diff
@@ -71,11 +102,14 @@
             My Recent <br />
             Works.
           </div>
-
-          <div><img src="assests/images/Indicator.svg" alt="Indicator" /></div>
+          <div class="indicator">
+            <img src="assests/images/Indicator.svg" alt="Indicator" />
+          </div>
         </div>
         <div class="project1">
-          <div class="project1-img"></div>
+          <div class="project1-img">
+            <img src="assests/images/project1-img.svg" alt="project1" />
+          </div>
           <div class="project1-content">
             <div class="project1-content-sec1">
               Multi-Post Stories <br />
```

Comment:
This is a project title so kindly use a header tag such as h2 - h6 to make your work semantically correct. 😉